bin/build-plugin-zip.sh

```diff
@@ -109,7 +109,7 @@ npm run build
 php bin/generate-gutenberg-php.php > gutenberg.tmp.php
 mv gutenberg.tmp.php gutenberg.php
 
-build_files=$(ls build/*/*.{js,css,asset.php} build/block-library/blocks/*.php)
+build_files=$(ls build/*/*.{js,json,css,asset.php} build/block-library/blocks/*.php)
```

Comment:
I was looking at a fix for this and went with this instead, which is a bit more specific:

```suggestion
build_files=$(ls build/*/*.{js,css,asset.php} build/block-library/blocks/*.{php,json})
```

Right now, it doesn't seem there are any other JSON files though:

```
$ find build -name '*.json'                           
build/block-library/blocks/shortcode.json
```